editor/code/syntaxes/coq.json

```diff
@@ -0,0 +1,229 @@
+{
+    "fileTypes": ["v"],
```

Comment:
Would be great if we could have some of this working for `.mv` files too, but I'm unsure how.

Note that we could also use LSP semantic highlighting.